docs/includes/eloquent-models/Planet.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace App\Models;
+
+use MongoDB\Laravel\Eloquent\Model;
+
+class Planet extends Model
```

Comment:
Can be written like this:
```suggestion
class Planet extends \MongoDB\Laravel\Eloquent\Model
```

Reply:
Good to know. Do you prefer showing the user this format instead? Or showing that it's another way to represent it?

Reply:
For this case, the current syntax is better.
```php
use MongoDB\Laravel\Eloquent\Model;

class Planet extends Model
```